index.js

```diff
@@ -25,7 +27,11 @@ class HomeBridgeTasmotaAirconHTTP {
     this.superagent = superagent; // TODO: Is superagent needed?
 
     this.fanSteps = config.fanSteps || 5; // Used to translate {0..100} to {1..fanSteps}
-    this.state = this._initialState(config);
+
+    // If exists, get Previous state from storage
+    if (api.user) storage.initSync({'dir': path.join(api.user.storagePath(), 'HomeBridgeTasmotaAirconHTTP')});
+    this.state = storage.getItemSync(this.name) || this._initialState(config);
```

Comment:
Let's say we have `{"foo":42}` stored on disk, but then we add a new key `{"bar":true}` to _initialState, then reading the data from disk will not have the "bar" key set, but instead it will be `uninitialized`. To fix this, the code could do this:

```
this.state = this._initialState(config); // `{"foo":100,"bar":true}`
const persistedState = storage.getItemSync(this.name) || {}; // `{"foo":42}`
Object.keys(persistedState).forEach(k => (this.state[k] = persistedState[k]));
// this.state.bar == true;
// this.state.foo == 42;
```

Reply:
Ok, I didn't think about evolutions of plugins, with new keys added or removed. I changed the code to handle it as you proposed.